.github/CODEOWNERS

```diff
@@ -1,5 +1,6 @@
 # Request reviews from the approvers team
 * @build-trust/approvers
+/.github/CONTRIBUTORS.csv @cla-pr-approvers
```

Comment:
I've added the team and given it write access to the repo, not sure why Github is still showing an error on this line.

Reply:
Haaaa! should be @build-trust/cla-pr-approvers 😅